### ISSUE TITLE
Deprecated functions, Rermeha failed

### ISSUE THREAD

**Issue by p2923260** [msvisser/remeha_home#58]
Logger: homeassistant.helpers.typing
Source: helpers/deprecation.py:206
First occurred: 7:47:36 PM (1 occurrences)
Last logged: 7:47:36 PM

HomeAssistantType was used from remeha_home, this is a deprecated alias which will be removed in HA Core 2025.5. Use homeassistant.core.HomeAssistant instead, please report it to the author of the 'remeha_home' custom integration

**Comment by m4v3r1ckNl** [comment]
Same issue:

Logger: homeassistant.helpers.typing
Source: helpers/deprecation.py:197
First occurred: 12:47:51 (1 occurrences)
Last logged: 12:47:51

HomeAssistantType was used from remeha_home, this is a deprecated alias which will be removed in HA Core 2025.5. Use homeassistant.core.HomeAssistant instead, please report it to the author of the 'remeha_home' custom integration

**Comment by jellewie** [comment]
Can confirm

```
Logger: homeassistant.helpers.typing
Source: helpers/deprecation.py:197
First occurred: 16:59:56 (1 occurrences)
Last logged: 16:59:56

HomeAssistantType was used from remeha_home, this is a deprecated alias which will be removed in HA Core 2025.5. Use homeassistant.core.HomeAssistant instead, please report it to the author of the 'remeha_home' custom integration
```

**Comment by msvisser** [comment]
Fixed with #59.
